generators/client/templates/angular/src/main/webapp/app/layouts/navbar/navbar.component.html.ejs

```diff
@@ -17,7 +17,7 @@
                 </a>
             </li>
             <!-- jhipster-needle-add-element-to-menu - JHipster will add new menu items here -->
-            <li *ngSwitchCase="true" ngbDropdown class="nav-item dropdown pointer" routerLinkActive="active" [routerLinkActiveOptions]="{exact: true}">
+            <li *ngSwitchCase="true" ngbDropdown class="nav-item dropdown pointer" display="dynamic" routerLinkActive="active" [routerLinkActiveOptions]="{exact: true}">
```

Comment:
Why is this necessary? I'm curious.

Reply:
@mraible I think it may have been a merge artifact? I was comparing with a recently-generated Boot JHipster app and those were in the diff. They aren't specifically required by the Micronaut version.